injectionapi/socketlabsclient.go

```diff
@@ -57,6 +66,7 @@ func CreateClientWithProxy(serverID int, apiKey string, proxyURL string) ISocket
 		APIKey:      apiKey,
 		EndpointURL: endpointURL,
 		ProxyURL:    proxyURL,
+		RequestTimeout: 120,
```

Comment:
Set the default value as a const like endpointURL on line 16

Reply:
added const